tests/wpt/run.py

```diff
@@ -32,7 +32,14 @@ def run_tests(paths=None, **kwargs):
     mozlog.commandline.log_formatters["servo"] = \
         (grouping_formatter.GroupingFormatter, "A grouping output formatter")
 
+    use_mach_logging = False
     if len(kwargs["test_list"]) == 1:
+        file_ext = os.path.splitext(kwargs["test_list"][0])[1].lower()
+        single_file_exts = frozenset([".htm", ".html", ".js", ".xhtml"])
+        if file_ext in single_file_exts:
```

Comment:
The usual obvious way is what I'd suggested. If there's no other compelling reason for frozenset, then we can simply use this.